extern/sector-storage/stores/remote.go

```diff
@@ -123,6 +149,12 @@ func (r *Remote) AcquireSector(ctx context.Context, s storage.SectorRef, existin
 	}
 	defer releaseStorage()
 
+	var iosize uint64 = 0
+	ssize, err := GetIoSizeByProofType(s.ProofType)
```

Comment:
I'd do `iosize = 32 << 10; if s.ProofType.SectorSize() > 8 << 20 {iosize=1<<20}`

Reply:
(this way we can drop GetIoSizeByProofType from above)